staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1679,6 +1679,34 @@ type VirtualMachineStatus struct {
 	// RunStrategy tracks the last recorded RunStrategy used by the VM.
 	// This is needed to correctly process the next strategy (for now only the RerunOnFailure)
 	RunStrategy VirtualMachineRunStrategy `json:"runStrategy,omitempty" optional:"true"`
+
+	// VolumeMigrationState contains the information about the volumes set
+	// updates related to the volumeUpdateStrategy
+	VolumeUpdateState *VolumeUpdateState `json:"volumeUpdateState,omitempty" optional:"true"`
```

Comment:
Do we need to have separate lifecycle for this field or can we embed it to the migration state?

Reply:
we want to keep the `VolumeUpdateState` and the ``VolumeMigrationState` distinct because eventually the  `VolumeUpdateState`  will also contain the information about declarative hotplug.